microcosm_flask/swagger/schemas.py

```diff
@@ -86,18 +92,21 @@ def iter_schemas(self, schema: Schema) -> Iterable[Tuple[str, Any]]:
         if not schema:
             return
 
+        if type(schema) in self.seen_schemas:
+            return
+
+        self.seen_schemas.append(type(schema))
+
         yield self.to_tuple(schema)
 
         for associated_schema in getattr(schema, associated_schemas_attr_name(schema.__class__), {}).values():
             yield self.to_tuple(associated_schema())
 
         for name, field in self.iter_fields(schema):
             if isinstance(field, Nested):
-                yield self.to_tuple(field.schema)
                 yield from self.iter_schemas(field.schema)
 
             if isinstance(field, List) and isinstance(field.inner, Nested):
-                yield self.to_tuple(field.inner.schema)
```

Comment:
Somewhat surprisingly, this line and the one above aren't necessary, and in fact were resulting in every nested schema being output twice!  Was being masked by the fact that the schemas are just put [directly into a dict](https://github.com/globality-corp/microcosm-flask/blob/a36f8372caa87b1b40e92b0a297789c7c0289c84/microcosm_flask/swagger/definitions.py#L146).  Caught this issue when adding one final sanity check unit test cc/ @GabeLambda @rh-glob @afallou

Reply:
Oops.